services/userService.js

```diff
@@ -0,0 +1,27 @@
+const bcrypt = require("bcrypt");
+const response = require("express");
+const jwt = require("jsonwebtoken");
+
+const userDao = require("../models/userDao");
+
+const signIn = async (email, password) => {
+  const hashedPassword = await userDao.getHashedPassword(email);
+  const compare = await bcrypt.compare(password, hashedPassword);
+
+  if (!compare) {
+    const err = new Error("PASSWORD_DOES_NOT_MATCH");
+    err.statusCode = 401;
+    throw err;
+  }
+
+  const userId = await userDao.getUserId(email);
+
+  const payLoad = userId;
+  const jwtToken = jwt.sign(payLoad, process.env.secretKey);
+
+  return jwtToken;
```

Comment:
```suggestion
  return jwt.sign(userId, process.env.secretKey);
```

세줄로 쓰지 않고 간단하게 쓸 수 있을 것 같습니다 :)

Reply:
payLoad를 띠로 변수로 지정해줘서 값을 할당해야하는줄 알았습니다! 저렇게 간단하게 쓰는 방법도 있었군요!